test/e2e-tests.sh

```diff
@@ -22,16 +22,34 @@ failed=0
 (( !failed )) && logger.success 'Cluster prepared for testing.'
 
 # Run serverless-operator specific tests.
-(( !failed )) && run_e2e_tests || failed=4
+if (( TEST_SCOPE_SERVERLESS_OPERATOR & TEST_SCOPE )); then
```

Comment:
Why not create two files at this point?

Reply:
Maybe. But, in this way, by default, all tests are executed. In the same time I was able to easily run just eventing tests using `0x14` mask.

Also, we can use that scope to split execution to more separate GitHub checks, by defining a environment variable, as I proposed on Slack.

Reply:
Can we please not do bitmasks here? That feels awfully complex for the task at hand. I might be the only one though...

Reply:
I feel the same. Let's please please not make this complexer than it needs to be. 

I'd suggest we do a simple, easy PR and get the tests execuded and passing.